test/test_coverage.sh

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env bash
+
+# Create a link for the code to current dir
+ln -s src/main/scala/org ./
```

Comment:
I see. Then, let's just rename the script to `submit_coverage.sh` or so to make it clear.